scss/atoms/buttons/_buttons.scss

```diff
@@ -0,0 +1,89 @@
+/*
+* ==========================================================================
+* Buttons
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis",
+	"mixins";
+
+.btn {
+	border: 1px solid transparent;
+	cursor: pointer;
+	letter-spacing: map-get($ui-btn, letter-spacing);
+	font-family: $ui-btn-font;
+	font-weight: map-get($ui-btn, font-weight);
+	margin: map-get($ui-btn, margin);
+	text-decoration: none;
+	text-transform: map-get($ui-btn, text-transform);
+	@include btn-colors($ui-btn-default);
+	@include btn-size($ui-btn-md);
+}
+
+.btn-group {
```

Comment:
I agree we probably shouldn't rely on cascading markup. Normally in a scenario like this I wouldn't think twice about writing it that way but sense we are writing this with a focus on being easy to override we may want to stick to the one class approach.

To avoid the too many classes problem maybe we could write a mixin for the main button styles and then include it on the .btn and the .btn-group--btn classes? Not sure if that's overkill but it's just a thought.